webapp-django/crashstats/crashstats/views.py

```diff
@@ -1143,6 +1151,79 @@ def crashes_per_user(request, default_context=None):
         count = group['adi_count']
         adi_by_version[version][date] = [count, throttle]
 
+    # Now, remember how we exploded certain versions into lists of
+    # versions. That means we get ADI counts like this:
+    #
+    #  version 1.1 : 1234,
+    #  version 1.2 : 3456,
+    #  version 1.3 : 1010
+    #
+    # But no number of "1.x" if that was what exploded into 1.1, 1.2, 1.3.
+    # So we need to put that back and manually sum the counts. So that
+    # eventually we end up with this instead:
+    #
+    #  version 1.x : 5700,   (=1234+3456+1010 as per the example)
+    #
+    # Note that we assume the throttle to be the same.
+    #
+
+    for version, sub_versions in exploded_version_numbers.items():
```

Comment:
I don't like that you rely on the "official" version numbers. Super Search knows better than that, and sometimes even have version numbers that did not make it to our database yet. There's the ``b0`` exception as well (when we did not have data about a build at processing time). 

So I would instead go for a different approach: look at all the versions ending in ``b`` (e.g. ``18.0b``), and then go through all results, looking for version numbers starting with the same number (``18.0b``) and merge those together.